cli_tools/common/utils/daisy/daisy_utils.go

```diff
@@ -193,3 +196,18 @@ func PostProcessDErrorForNetworkFlag(action string, err error, network string, w
 		}
 	}
 }
+
+// RunWorkflowWithCancelSignal runs Daisy workflow with accepting Ctrl-C signal
+func RunWorkflowWithCancelSignal(ctx context.Context, w *daisy.Workflow) error {
+	c := make(chan os.Signal, 1)
+	signal.Notify(c, os.Interrupt)
```

Comment:
unit test?

Reply:
can't do unit test for this (unless we add some complex mechanism only for unit test) since it's hard to launch a real workflow and keep it executing on unit test env. It's covered by e2e.